src/rgw/rgw_lc_s3.cc

```diff
@@ -0,0 +1,112 @@
+#include <string.h>

```

Comment:
This file is formatted with DOS line endings.  It should be converted to UNIX line endings (dos2unix can do it)